.github/workflows/ci_build.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+        server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
+        settings-path: ${{ github.workspace }} # location for the settings.xml file
+
+    - name: Setup git
+      run: |
+        git config --global user.email "actions@github.com"
+        git config --global user.name "github-actions"
+
+    - name: Build with Maven
+      run: ./paper jar
+
+    - name: Upload Build Artifact
```

Comment:
This is nice, since we'll get the jar for all the PRs available and ready for testing. Though it'd be nice to have the jar attached directly to PR, if the build cause is a PR.

But I believe we also need a way to version it in a way, that allows for use by API. Through that can be added later, if we use Jenkins for releases and GA just to build stuff always.